components/hooks/post/postHooks.ts

```diff
@@ -125,19 +135,14 @@ const usePost = (commentId: string | null): UsePost => {
         post_id: currentPost.post.id,
         max_depth: 10,
         type_: "All",
-        sort: "Top",
+        sort: commentsSort,
         parent_id:
           commentId && !ignoreCommentId ? Number(commentId) : undefined,
       });
 
-      const ordered = commentsRes.comments.sort((a, b) =>
```

Comment:
Deleting this logic feels wrong... I'm not quite sure what it was doing, but it seemed to have been preventing the sort on line 128 from happening